packages/cursorless-org-docs/src/docs/user/release-notes/0.28.0.md

```diff
@@ -1,10 +1,13 @@
 ---
 version: "0.28.0"
-releaseDate: "2023-09-01"
+release_date: "2023-09-01"
+sidebar_position: -2
```

Comment:
using negative numbers here so that we just keep increasing the size and newer release notes will appear at the top